pkg/webhook/validating/network.go

```diff
@@ -124,5 +124,16 @@ func NetworkDeleteValidation(ctx context.Context, req *admission.Request, handle
 			return admission.Denied(fmt.Sprintf("still have child subnet %s", subnet.Name))
 		}
 	}
+
+	if network.Spec.Type == ramav1.NetworkTypeOverlay && feature.MultiClusterEnabled() {
+		remoteClusterLister := &ramav1.RemoteClusterList{}
```

Comment:
Lister has a special meaning in k8s, `remoteClusterLister` should rename to `remoteClusterList`